<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidgetClass</class>
 <widget class="QWidget" name="MainWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="url"/>
     </item>
     <item>
      <widget class="QPushButton" name="selectBtn">
       <property name="text">
        <string>选择</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startBtn">
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pauseBtn">
       <property name="text">
        <string>暂停</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="4">
       <widget class="PlayImage" name="playImage_2" native="true"/>
      </item>
      <item row="1" column="4">
       <widget class="PlayImage" name="playImage_5" native="true"/>
      </item>
      <item row="1" column="3">
       <widget class="PlayImage" name="playImage_4" native="true"/>
      </item>
      <item row="2" column="3">
       <widget class="PlayImage" name="playImage_7" native="true"/>
      </item>
      <item row="0" column="3">
       <widget class="PlayImage" name="playImage_1" native="true"/>
      </item>
      <item row="2" column="4">
       <widget class="PlayImage" name="playImage_8" native="true"/>
      </item>
      <item row="2" column="2">
       <widget class="PlayImage" name="playImage_6" native="true"/>
      </item>
      <item row="1" column="2">
       <widget class="PlayImage" name="playImage_3" native="true"/>
      </item>
      <item row="0" column="2">
       <widget class="PlayImage" name="playImage" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PlayImage</class>
   <extends>QWidget</extends>
   <header>playimage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MainWidget.qrc"/>
 </resources>
 <connections/>
</ui>
